web/elm/src/Build/StepTree/StepTree.elm

```diff
@@ -722,8 +722,31 @@ viewMetadata =
                 ]
             )
         )
-        >> Dict.fromList
-        >> DictView.view []
+        >> List.map
+            (\( key, value ) ->
+                Html.tr []
+                    [ Html.td
+                        [ style "text-align" "left"
+                        , style "vertical-align" "top"
+                        , style "background-color" "rgb(45,45,45)"
+                        , style "border-bottom" "5px solid rgb(45,45,45)"
+                        , style "padding" "5px"
+                        ]
+                        [ key ]
+                    , Html.td
+                        [ style "text-align" "left"
+                        , style "vertical-align" "top"
+                        , style "background-color" "rgb(35,35,35)"
+                        , style "border-bottom" "5px solid rgb(45,45,45)"
+                        , style "padding" "5px"
```

Comment:
I'd really like to see a type like `type MetadataCellType = Key | Value` and a function `metadataCell : MetadataCellType -> List Html.Attribute` in `Build/Styles.elm` and then have these two `Html.td` calls look like

```
[ Html.td (Styles.metadataCell Styles.Key) [ key ]
, Html.td (Styles.metadataCell Styles.Value) [ value ]
]
```

I think this would strongly ease the flow of reading.

Reply:
great suggestion, you must be good at this